src/Raven.Studio/typescript/commands/licensing/getConnectivityToLicenseServerCommand.ts

```diff
@@ -0,0 +1,16 @@
+﻿import commandBase = require("commands/commandBase");
+import endpoints = require("endpoints");
+
+class getConnectivityToLicenseServerCommand extends commandBase {
+ 
+    execute(): JQueryPromise<Raven.Server.Web.Studio.LicenseHandler.ConnectivityToLicenseServer> {
+        const url = endpoints.global.license.licenseServerConnectivity;
+
+        return this.query<any>(url, null)
```

Comment:
don't we need to pass datatype: undefined? IIRC endpoint results empty result. Please check if it doens't return empty string along with header: `applicatoin/json`.

Reply:
leaving this - as discussed